src/qiskit_iqm/iqm_backend.py

```diff
@@ -106,6 +113,14 @@ def close_client(self):
             self.client.close_auth_session()
         self.client = None
 
+    def qubit_name_to_index(self, name: str) -> int:
+        """Given an IQM-style qubit name ('QB1', 'QB2', etc.) return the corresponding index in the register."""
+        return self._qb_to_idx[name]
```

Comment:
I understand that, by design, this should never fail, but I instinctively have a bad feeling about optimistically retrieving a value like this. I don't have good suggestions though :D

Perhaps there is a suitable exception it can throw in case key is not found?

Reply:
Do you mean scenarios like a user requesting an incorrect qubit name? I guess I can return `None` in that case

Reply:
Done